__tests__/app.test.js

```diff
@@ -134,7 +134,12 @@ describe("GET /api/:article_id/comments", () => {
   });
 
   test("Should return 404 if given a valid format id that doesn't exist in comments database", () => {
-    return request(app).get("/api/99/comments").expect(404);
+    return request(app)
+      .get("/api/99/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body).toEqual({ msg: "bad request" });
```

Comment:
the message associated with a 404 is "not found" - bad request would be for a 400